src/frameworks/vite.json

```diff
@@ -4,7 +4,7 @@
   "category": "build_tool",
   "detect": {
     "npmDependencies": ["vite"],
-    "excludedNpmDependencies": ["@shopify/hydrogen", "@builder.io/qwik", "solid-start", "solid-js"],
+    "excludedNpmDependencies": ["@shopify/hydrogen", "@builder.io/qwik", "solid-start", "solid-js", "@sveltejs/kit"],
```

Comment:
Adding SvelteKit as an excluded dependency so it doesn't get picked up by the vite config.